### PR TITLE
Add Usb discovery

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add support for USB discovery.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54904
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/usb.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - Utility
+ha_iot_class: Local Push
+ha_release: 2021.9
+ha_domain: usb
+ha_quality_scale: internal
+ha_codeowners:
+  - '@bdraco'
+---
+
+The `usb` "USB Discovery" integration will watch the host for USB devices. Discovered integrations will show up in the discovered section on the integrations page in the configuration panel.
```
Comment (frenck) [inline source/_integrations/usb.markdown]:
```suggestion
The USB Discovery integration will watch the host for USB devices. Discovered integrations will show up in the discovered section on the integrations page in the configuration panel.
```

File: source/_integrations/usb.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- On Linux, devices are detected as soon as they are plugged in.
+
+- On Windows and Mac OS, devices are detected at startup and hourly.
+
+## Configuration
+
+This integration is by default enabled, unless you've disabled or removed the [`default_config:`](/integrations/default_config/) line from your configuration. If that is the case, the following example shows you how to enable this integration manually:
```
Comment (frenck) [inline source/_integrations/usb.markdown:22]: This means this PR should update the list on this page as well:

<https://www.home-assistant.io/integrations/default_config>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/usb.markdown (hunk 2 above)
Comment (bdraco) [reply source/_integrations/usb.markdown:22]: Good call :)

**Review by frenck — APPROVED** [review]
Thanks, @bdraco 👍

**Pull request merged**